pandas/tests/arrays/interval/test_interval.py

```diff
@@ -26,22 +26,6 @@ def left_right_dtypes(request):
 
 class TestMethods(object):
 
-    @pytest.mark.parametrize('repeats', [0, 1, 5])
-    def test_repeat(self, left_right_dtypes, repeats):
```

Comment:
deleted as these appear to be testing the same thing as the base class tests in `tests/extension/base/methods.py`